buildutils/torque.spec.in

```diff
@@ -61,9 +61,9 @@
 %define ac_trqauthd_sock_dir %{?trqauthd_sock_dir:%{trqauthd_sock_dir}}%{!?trqauthd_sock_dir:/tmp}
 
 ### Build Requirements
-%define breq_cpuset   %{?with_cpuset:hwloc-devel}
+%define breq_cpuset   %{!?hwloc_path:%{?with_cpuset:hwloc-devel}}
 %define breq_gui      %{?with_gui:tcl-devel tk-devel tclx}
-%define breq_numa     %{?with_cpuset:hwloc-devel}
```

Comment:
The dependency is at build time, not install time. This dependency exists because you need hwloc.h etc when compiling torque. This is provided by hwloc-devel or via --with-hwloc-path.